eng/Packages.Data.props

```diff
@@ -163,43 +111,101 @@
     <PackageReference Update="Microsoft.Extensions.Options" Version="2.1.0" />
   </ItemGroup>
 
-  <!-- Test only packages -->
-  <ItemGroup Condition="('$(IsTestProject)' == 'true') OR ('$(IsTestSupportProject)' == 'true')">
-
-    <PackageReference Update="Microsoft.CSharp" Version="4.6" />
+  <!--
+    Build time packages
+    All should have PrivateAssets="All" set so they don't become pacakge dependencies
+  -->
+  <ItemGroup>
+    <PackageReference Update="AutoRest.CSharp.V3" Version="1.0.0-alpha.20201203.2" PrivateAssets="All" />
+    <PackageReference Update="Azure.ClientSdk.Analyzers" Version="0.1.1-dev.20200929.2" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.DotNet.GenAPI" Version="5.0.0-beta.19552.1" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Update="StyleCop.Analyzers" Version="1.1.118" PrivateAssets="All" />
+    <PackageReference Update="coverlet.collector" Version="1.3.0" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.1" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.DotNet.ApiCompat" Version="5.0.0-beta.20467.1" PrivateAssets="All" />
+    <PackageReference Update="SauceControl.InheritDoc" Version="1.2.0" PrivateAssets="All" />
```

Comment:
Should we further split this into MS and non-MS? Or have this list sorted to make it easier to find the non-ms dependencies?

Reply:
Sorted